src/iv/imageviewer.h

```diff
@@ -240,6 +244,19 @@ class ImageViewer final : public QMainWindow {
     void rawcolor(bool val) { m_rawcolor = val; }
     bool rawcolor() const { return m_rawcolor; }
 
+#ifdef USE_OCIO
+    enum OCIO_OPTIMIZATION { OCIO_OPTIMIZATION_NONE      = 0,
```

Comment:
The OCIO UX working group recommendation is NOT to expose optimization settings to end users.  It is very rarely needed and just causes confusion for end-users.  (I know it is in ociodisplay, we should remove it perhaps, but that is more of a debugging tool.)

Reply:
Thanks Doug, I have removed that.